webproject/src/main/webapp/my-rooms.html

```diff
@@ -137,6 +137,7 @@ <h1 class="mb-0 site-logo">
     <script src="js/main.js"></script>
     <script src="js/myRooms.js"></script>
     <script src="js/logout.js"></script>
+    <script src="js/redirect.js"></script>
```

Comment:
is this file exists?

Reply:
No, it does not! I did try to delete all of the references but it seems I missed one.